specs/lib/stalePullRequestSpec.js

```diff
@@ -144,11 +167,51 @@ describe('stalePullRequest', function () {
             .catch(done.fail);
     });
 
+    it('stalePullRequest._processPullRequest does not post if there is a recent commit', function (done) {
+        var repositorySettings = new RepositorySettings();
+        var commentsUrl = 'https://url';
+        var commitsUrl = 'https://commits';
+        var pullRequest = {
+            comments_url: commentsUrl,
+            commits_url: commitsUrl,
+            user: {login: 'boomerjones'}
+        };
+
+        spyOn(requestPromise, 'get').and.callFake(function (options) {
+            if (options.resolveWithFullResponse === true) {
+                return Promise.resolve({headers: {link: '<https://url?page=2>; rel="next",<https://url?page=3>; rel="last"'}});
+            } else if (options.url === commentsUrl + '?page=3') {
+                var timestamp = new Date(Date.now());
+                timestamp.setDate(timestamp.getDate() - repositorySettings.maxDaysSinceUpdate);
+                return Promise.resolve([{
+                    updated_at: timestamp,
+                    user: {login: 'boomerjones'}
+                }]);
+            } else if (options.url === commitsUrl) {
+                return Promise.resolve(commitsData);
+            }
+            return Promise.reject(new Error('Unexpected Url: ' + options.url));
+        });
+        spyOn(requestPromise, 'post');
```

Comment:
Is this spy needed? I don't see a `callFake` or an expectation that this is called.

Reply:
I need to create a spy in order for the following to work right? (a few lines below):

```
expect(requestPromise.post).not.toHaveBeenCalled();
```

Reply:
Missed the expect, the way you have it is good. :+1: